src/app/files.service.ts

```diff
@@ -161,11 +161,11 @@ export class FilesService {
                                       filamentWeight: this.service.convertFilamentLengthToAmount(filamentLength),
                                   }
                                 : {}),
-                            thumbnail: data.path.endsWith('.ufp.gcode')
+                            thumbnail: data.thumbnail
                                 ? this.configService
-                                      .getURL('plugin/UltimakerFormatPackage/thumbnail/')
-                                      .replace('/api/', '/') + data.path.replace('.ufp.gcode', '.png')
-                                : undefined,
+                                      .getURL('' + data.thumbnail + '')
```

Comment:
```suggestion
                                      .getURL(data.thumbnail)
```

I think we don't need the concatenation anymore.

Reply:
I initially tried this but was getting errors, but I think that's because of `[string]`, will update and verify.